property.go

```diff
@@ -87,3 +114,82 @@ func (client *Client) DeletePropertyDefinition(input string) error {
 	err := client.Mutate(&m, v, WithName("PropertyDefinitionDelete"))
 	return HandleErrors(err, m.Payload.Errors)
 }
+
+func (client *Client) GetProperty(owner string, definition string) (*Property, error) {
+	var q struct {
+		Account struct {
+			Property Property `graphql:"property(owner: $owner, definition: $definition)"`
+		}
+	}
+	v := PayloadVariables{
+		"owner":      *NewIdentifier(owner),
+		"definition": *NewIdentifier(definition),
+	}
+	err := client.Query(&q, v, WithName("PropertyGet"))
+	if q.Account.Property.Definition.Id == nil {
+		err = fmt.Errorf("Property with ID or alias matching '%s' on Service with ID or alias matching '%s' not found", owner, definition)
```

Comment:
this error message is not correct / the string template vars are backwards i think.

Reply:
They are backwards, going to correct this.